app/src/main/res/values/strings.xml

```diff
@@ -217,4 +213,40 @@
      When the described button is pressed, the overlay will open and the browser will be hidden underneath it -->
     <string name="hint_press_menu_to_open_overlay_a11y">Press the menu button for %1$s Home, search, settings, and more</string>
 
+    <!-- FxA Strings-->
+    <!-- Content description for a button that will prompt a user to sign in, in order to receive tabs pushed from FF
+    on another device -->
+    <string name="fxa_login_a11y">Receive tabs</string>
+    <!-- Heading for settings menu item button and CTA link on new update -->
```

Comment:
I'm not sure this description is super helpful to localizers: what's a CTA link? What's a new update? What happens when the button is pressed? Also, "Heading" isn't quite accurate: it's more of a button description. I'm thinking something like:

"Description for Receive Tab buttons (in navigation toolbar & settings, both on the homescreen) that will launch a Firefox Account sign-in experience so users can receive desktop Firefox tabs on their Fire TV".